actors/builtin/miner/miner_test.go

```diff
@@ -1250,9 +1257,11 @@ func (h *actorHarness) extendSector(rt *mock.Runtime, sector *miner.SectorOnChai
 	rt.ExpectValidateCallerAddr(h.worker)
 
 	st := getState(rt)
+	info, err := st.GetInfo(rt.AdtStore())
+	require.NoError(h.t, err)
 	newSector := *sector
 	newSector.Expiration += extension
-	qaDelta := big.Sub(miner.QAPowerForSector(st.Info.SectorSize, &newSector), miner.QAPowerForSector(st.Info.SectorSize, sector))
+	qaDelta := big.Sub(miner.QAPowerForSector(info.SectorSize, &newSector), miner.QAPowerForSector(info.SectorSize, sector))
```

Comment:
As for partition size, we need sectorSize enough that the harness should expose it directly.